themes/SuiteP/css/suitep-base/suitepicon.html

```diff
@@ -1,216 +0,0 @@
-<!DOCTYPE html>
```

Comment:
This file removed but I can not find any reference for it in your PR. Please make sure each reference for removed files also removed from code base.

Reply:
I have added this file. It provides a preview for all the suitepicon classes